db/migrations/20210308172740_createClucks.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable('clucks', table => {
+    table.bigIncrements('id')
+    table.string('username')
+    table.string('image_url')
```

Comment:
type `string` for `image_url` limits it to only 255 characters, but some urls are longer